tests/test_math.py

```diff
@@ -73,33 +73,33 @@ def testSigmoid(self):
 class MatmulTests(unittest.TestCase):
     """Matmul Tests"""
 
-    def testMatmul1DInt(self):
+    def test_mat_mul_1d_int(self):
         t1 = TensorBase(np.array([1, 2, 3]))
         t2 = TensorBase(np.array([4, 5, 6]))
         self.assertEqual(syft.matmul(t1, t2), syft.dot(t1, t2))
 
-    def testMatmul1DFloat(self):
+    def test_mat_mul_1d_float(self):
         t1 = TensorBase(np.array([1.3, 2.5, 3.7]))
         t2 = TensorBase(np.array([4.9, 5.8, 6.5]))
         self.assertEqual(syft.matmul(t1, t2), syft.dot(t1, t2))
 
-    def testMatmul2DIdentity(self):
+    def test_mat_mul_2d_identity(self):
         t1 = TensorBase(np.array([[1, 0],
                                   [0, 1]]))
         t2 = TensorBase(np.array([[5.8, 6.5],
                                   [7.8, 8.9]]))
         self.assertTrue(syft.equal(syft.matmul(t1, t2), [[5.8, 6.5],
                                                          [7.8, 8.9]]))
 
-    def testMatmul2DInt(self):
+    def test_mat_mul_2d_int(self):
         t1 = TensorBase(np.array([[1, 2],
                                   [3, 4]]))
         t2 = TensorBase(np.array([[5, 6],
                                   [7, 8]]))
         self.assertTrue(syft.equal(syft.matmul(t1, t2), [[19, 22],
                                                          [43, 50]]))
 
-    def testMatmul2DFloat(self):
+    def test_mat_mul_2d_float(self):
```

Comment:
:+1: to the idea. Just a tiny comment, I think the MatmulTests should be `test_matmul_X` and not `test_mat_mul_X`.

Reply:
yes. I agree 👍